docs/LibraryEvolution.rst

```diff
@@ -578,7 +578,9 @@ There are very few safe changes to make to protocols and their members:
 - The ``@discardableResult`` and ``@warn_unqualified_access`` attributes may
   be added to or removed from a function requirement.
 - A new ``associatedtype`` requirement may be added (with the appropriate
-  availability), as long as it has a default implementation.
+  availability), as long as it has a default implementation. If the protocol
+  did not have one or more ``associatedtype`` requirements before the change,
+  then this is a `binary-compatible source-breaking change`.
```

Comment:
Should these be double-backticks?

Reply:
I think this is right, double backticks are for formatting code, single backticks are for references. Elsewhere in this document:

https://github.com/apple/swift/blob/a0cdeb5fecf30cfb01d31b5b8a4360851bdac0b9/docs/LibraryEvolution.rst#L88-L89

https://github.com/apple/swift/blob/a0cdeb5fecf30cfb01d31b5b8a4360851bdac0b9/docs/LibraryEvolution.rst#L162-L163